docs/how-to/AdjustingDockerfile.md

```diff
@@ -0,0 +1,14 @@
+# Customizing the base docker image (mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210922.v1) 
```

Comment:
Can we specify somehow that we are talking about environments to be used in training/inference pipelines?
Customizing a docker image is something that could also apply to webservices deployment, but that would be done differently, not via `configuration/environments/` (issue #29 is tracking this enhancement).